cli/src/commands/rename.ts

```diff
@@ -0,0 +1,72 @@
+import { CLIError } from '@oclif/errors';
+import Command from '../utils/command';
+import flags from '../utils/cli-flags';
+import {
+  getProjectRootDir,
+  findProjectByDirectory,
+} from '../utils/helpers';
+
+export default class Rename extends Command {
+  public static description = 'rename an application';
+  public static examples = [
+    `$ ${Command.cliBinName} rename great-rhinoceros-42 surpassing-lemur-17`,
```

Comment:
This help doesn't give the direction; consider adding some help text to this example.  Assembly language ha{d,s} this problem with AT&T (?) syntax, best be sure.  Actually I'm not even sure of the relationship this arg has with the `-s` flag argument, and O'Clif is too confusing for me to find these 2 positional arguments.